sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/one/Extensions.java

```diff
@@ -328,7 +370,8 @@ public boolean equals(Object o) {
         if (app != null ? !app.equals(that.app) : that.app != null) return false;
         if (net != null ? !net.equals(that.net) : that.net != null) return false;
         if (sdk != null ? !sdk.equals(that.sdk) : that.sdk != null) return false;
-        return loc != null ? loc.equals(that.loc) : that.loc == null;
+        if (loc != null ? !loc.equals(that.loc) : that.loc != null) return false;
```

Comment:
regenerate equals/hashcode after reordering

Reply:
We should keep in mind that generated methods like `hashCode` and `equals` shouldn't be manually modified. Remove those methods and regenerate.